src/Middleware/WebSockets/src/WebSocketMiddleware.cs

```diff
@@ -46,6 +56,12 @@ public WebSocketMiddleware(RequestDelegate next, IOptions<WebSocketOptions> opti
             // TODO: validate options.
         }
 
+        /// <summary>
+        /// Processes a request to determine if it is a WebSocket request, and if so,
+        /// sets the <see cref="IHttpWebSocketFeature"/> on the <see cref="HttpContext.Features"/>.
+        /// </summary>
+        /// <param name="context">The <see cref="HttpContext"/> representing the request.</param>
+        /// <returns>The <see cref="Task"/> that represents the completion of the middleware pipeline.</returns>
```

Comment:
Elsewhere we've removed `returns` for async Task APIs and only included it when there was a value returned.